### DISCUSSION TITLE
Spotify long song names

### DISCUSSION THREAD

**Discussion by Xiaorui-Huang** [JanDeDobbeleer/oh-my-posh#1755] [Troubleshoot] [answered]
Is it possible to compare string length in the if statement to cap the maximum number of characters displayed? Or perhaps something like substring?

**Comment by JanDeDobbeleer** [comment, marked as answer]
Yes, you can make use of the [trunc](https://github.com/Masterminds/sprig/blob/master/docs/strings.md#trunc) function in your template: `{{ .Icon }}{{ if ne .Status \"stopped\" }}{{ .Artist }} - {{ trunc 10 .Track }}{{ end }}`
